cra/task-com-editor.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cra-comm-editor
+spec:
+  params:
+    - name: repository
+      description: the git repo url
+    - name: pr-url
+      description: merge request url
+      default: ""
+    - name: project-id
+      description: project id
+      default: ""
+    - name: comment-fp
+      description: comments filepath
+    - name: scm-type
+      description: source code type used (github, github-ent, gitlab)
+  steps:
+    - name: comm-editor
+      image: icr.io/continuous-delivery/cra-comm-editor:main.1111
+      imagePullPolicy: Always
+      workingDir: "/artifacts"
+      command: ["/bin/sh", "-c"]
+      args:
+        - |
+          #!/bin/sh
+          GIT_PROPS_FILE=./build.properties
```

Comment:
Where does this file(name) `build.properties` come form ?
It is weird that this properties filename that should have been created by a previous task execution in the pipeline can not be pararemterized/configurable